package/utils/px5g-wolfssl/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2020 Paul Spooren <mail@aparcar.org>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=px5g-wolfssl
+PKG_RELEASE:=1
+PKG_LICENSE:=LGPL-2.1
```

Comment:
erm LGPL is for libraries. This is a utility.

Reply:
Updated to GPL-2.0-or-later